src/faultlog/service/faultlog_bmcboot.service.in

```diff
@@ -0,0 +1,24 @@
+[Unit]
+Description=PowerVM Handler Host
+Wants=xyz.openbmc_project.Dump.Manager.service
+After=xyz.openbmc_project.Dump.Manager.service
+Wants=mapper-wait@-xyz-openbmc_project-inventory.service
+After=mapper-wait@-xyz-openbmc_project-inventory.service
+Wants=org.open_power.HardwareIsolation.service
+After=org.open_power.HardwareIsolation.service
+Wants=mapper-wait@-xyz-openbmc_project-logging.service
+After=mapper-wait@-xyz-openbmc_project-logging.service
+Wants=com.ibm.VPD.Manager.service
+After=com.ibm.VPD.Manager.service
+
+Wants=obmc-power-on@%0.target
+After=obmc-power-on@%0.target
+Conflicts=obmc-power-off@%0.target
```

Comment:
This service should skip in a normal boot and execute only in reboot, at least NAG error should not be logged during a normal BMC boot